tests/foreman/api/test_repository.py

```diff
@@ -1295,6 +1295,42 @@ def test_positive_sync_with_treeinfo_ignore(
             f'{repo.full_path}.treeinfo'
         ), 'The treeinfo file is missing in the KS repo but it should be there.'
 
+    @pytest.mark.tier3
+    @pytest.mark.parametrize('client_repo', ['rhsclient7', 'rhsclient8', 'rhsclient9'])
+    def test_positive_katello_agent_availability(
+        self, target_sat, function_sca_manifest_org, client_repo
+    ):
+        """Verify katello-agent package remains available in the RH Satellite Client repos
+        for older Satellite versions (Sat 6.14 is the last one supporting it).
+
+        :id: cd5b4e5b-e4f1-4d00-b171-30cd5b1e7ce8
+
+        :parametrized: yes
+
+        :steps:
+            1. Enable RH Satellite Client repo and sync it.
+            2. Read the repo packages.
+
+        :expectedresults:
+            1. Katello-agent remains available in the Client repos.
```

Comment:
Also, I know client repo is common to all  .z branches, but shouldn't we verify if this exists there too? can you update labels accordingly

Reply:
Well, my idea was to test it just until `6.14.z` (the last version using katello-agent) is supported/tested. I expect `6.14.z` releases be more frequent than the former z-releases, so I think we are good with 6.14.z.